### PR TITLE
bpo-34443: Use __qualname__ instead of __name__ in enum exception messages.

### PR DESCRIPTION
**Pull request by doerwalter**
https://bugs.python.org/issue34443

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2019-07-17-11-10-08.bpo-34443.OFnGqz.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Exceptions from :mod:`enum` now use the `__qualname` of the enum class in
```
Comment (asottile) [inline Misc/NEWS.d/next/Library/2019-07-17-11-10-08.bpo-34443.OFnGqz.rst]:
```suggestion
Exceptions from :mod:`enum` now use the ``__qualname__`` of the enum class in
```

File: Misc/NEWS.d/next/Library/2019-07-17-11-10-08.bpo-34443.OFnGqz.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+the exception message instead of the `__name__`.
```
Comment (asottile) [inline Misc/NEWS.d/next/Library/2019-07-17-11-10-08.bpo-34443.OFnGqz.rst]:
```suggestion
the exception message instead of the ``__name__``.
```

**Review by asottile — APPROVED** [review]
otherwise looks good

**Comment by doerwalter** [comment]
OK, I've fixed the ReST markup.

**Review by ethanfurman — APPROVED** [review]

**Pull request merged**